### PR TITLE
LPS-139170 The boolean field values are not counting as one in form entries (Summary) when form has translations

### PR DESCRIPTION
**Pull request by mateusgomes01**
Related Issue: https://issues.liferay.com/browse/LPS-139170

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxDDMFormFieldTypeReportProcessor.java [hunk 1]
```diff
@@ -46,12 +45,10 @@ public JSONObject process(
...
-					value.getDefaultLocale(), "true");
+				valueString = Boolean.TRUE.toString();
 			}
 			else {
-				valueString = LanguageUtil.get(
-					value.getDefaultLocale(), "false");
+				valueString = Boolean.FALSE.toString();
 			}
 
 			updateData(
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxDDMFormFieldTypeReportProcessor.java:54]: @mateusgomes01 can you please check if we really need this if/else. I believe we can simply do:
```
if (Validator.isNotNull(valueString)) {
	updateData(
		ddmFormInstanceReportEvent,
		fieldJSONObject.getJSONObject("values"), valueString);
}
```

**Comment by liferay-forms** [comment]
Hey @mateusgomes01, can you please check if these failures are related to your changes? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxDDMFormFieldTypeReportProcessor.java (hunk 1 above)
Comment (mateusgomes01) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxDDMFormFieldTypeReportProcessor.java:54]: I agree @carolmariaabb . I'll make the change. Thanks!

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by mateusgomes01** [comment]
ci:test:sf
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @mateusgomes01, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by mateusgomes01** [comment]
ci:reopen

**Pull request closed without merging**